config/routes.rb

```diff
@@ -1,5 +1,12 @@
 Rails.application.routes.draw do
-  root 'homes#index'
+  # root 'homes#index'
+  root "app#index"
```

Comment:
Let chat about this. Also, remove comment

Reply:
removed comment, left app#index to match controller